install/kubernetes/cilium/values.yaml.tmpl

```
@@ -1049,39 +1049,51 @@ hubble:
   socketPath: /var/run/cilium/hubble.sock
 
   # -- Configures the redact options for Hubble.
-  # Example:
-  #
-  #   redact:
-  #     enabled: true
-  #     http:
-  #       urlQuery: true
-  #     kafka:
-  #       apiKey: false
-  #
-  # You can specify the options from the helm CLI:
-  #
-  #   --set hubble.redact.enabled="true"
-  #   --set hubble.redact.http.urlQuery="true"
-  #   --set hubble.redact.kafka.apiKey="false"
   redact:
     enabled: false
     http:
+        # -- Configures the http urlQuery redact option.
+        # Example:
+        #
+        #   redact:
+        #     enabled: true
+        #     http:
+        #       urlQuery: true
+        #
+        # You can specify the options from the helm CLI:
+        #
+        #   --set hubble.redact.enabled="true"
+        #   --set hubble.redact.http.urlQuery="true"
         urlQuery: false
         # -- Configures the http headers redact allow/deny lists
+        # Example:
+        #   hubble.redact.http.allow:
+        #     - traceparent
+        #     - tracestate
+        #     - Cache-Control
+        #
+        # You can specify the options from the helm CLI:
+        #   --set hubble.redact.enabled="true"
+        #   --set hubble.redact.http.headers.allow="traceparent,tracestate,Cache-Control"
         headers:
-          # Example:
-          #   hubble.redact.http.allow:
-          #     - traceparent
-          #     - tracestate
-          #     - Cache-Control
-          #
-          # or by using helm CLI:
-          #   --set hubble.redact.http.headers.allow="traceparent,tracestate,Cache-Control"
           allow: []
           # A deny list for the https headers is supported (cannot be used along with the allow list):
+          #   --set hubble.redact.enabled="true"
           #   --set hubble.redact.http.headers.deny="Authorization,Proxy-Authorization"
```

Now that we split the docstrings to individual keys I think we can reintroduce the full `deny` example, likewise sticking to the same format as for other keys:
```suggestion
        # -- List of HTTP headers to deny -- matching headers will be redacted. Note: `allow` and `deny` lists cannot be used both at the same time, only one can be present.
        # Example:
        #   redact:
        #     enabled: true
        #     http:
        #       headers:
        #         deny:
        #           - Authorization
        #           - Proxy-Authorization
        #
        # You can specify the options from the helm CLI:
        #   --set hubble.redact.enabled="true"
        #   --set hubble.redact.http.headers.deny="Authorization,Proxy-Authorization"
```